DB/Item.lua

```diff
@@ -150,4 +150,17 @@ function Item:IsMount(entry)
 	return entry.type == CONSTANTS.ITEM_TYPES.MOUNT
 end
 
+function Item:HasZone(entry)
+	if entry.coords then
```

Comment:
I think early exit makes it more readable:

`if not ... then return false end`

or just

`if not ... then return end`

If you agree, could you please change that?

Reply:
I didn't think of that, but it does make sense. I've fixed that part in 039729e9df6633a37a838bf692b3abb8955a5b95.